Remove average command (#26)
AY2122S1-CS2103-F10-1/tp: docs/DeveloperGuide.md

**MSS**

1.  User requests to list applicants.
2.  MrTechRecruiter shows a list of applicants.

> 3.  User requests to delete a specific person in the list using the exact name.
> 4.  MrTechRecruiter deletes the person.

Reviewer: ```suggestion
3.  User requests to delete a specific applicant in the list using the exact name.
4.  MrTechRecruiter deletes the applicant.
```

**MSS**

1.  User requests to list applicants.
2.  MrTechRecruiter shows a list of applicants.
3.  User requests to delete a specific person in the list using the exact name.

> 4.  MrTechRecruiter deletes the person. <br>

Reviewer 2: I think double new lines can be used to start with a new line (consistent with other parts of the document)

> 1. User requests to see average rejection rates for all job positions.

Reviewer: ```suggestion
1. User requests to see a list of all job positions
```

**MSS**

> 1. User keys in the particular job position using the 'find' function.

Reviewer: ```suggestion
1. User searches for a particular job position using the 'find' function.
```
